Automatic Validation ended with:
> 2025-04-30T14:54:42.7004039Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[InstallationMetadata] Description: Value type not permitted by 'type' constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'InstallationMetadata'. ] File: JReleaser.jreleaser.installer.yaml Manifest Warning: Unknown field. [DefaultInstallLocation] Manifest Warning: Unknown field. [UpgradeCode]
 2025-04-30T14:54:42.7015940Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[InstallationMetadata] Description: Value type not permitted by 'type' constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'InstallationMetadata'. ] File: JReleaser.jreleaser.installer.yaml Manifest Warning: Unknown field. [DefaultInstallLocation] Manifest Warning: Unknown field. [UpgradeCode]

(Automated response - build 1048.)

--- a/manifests/j/JReleaser/jreleaser/1.18.0/JReleaser.jreleaser.installer.yaml
+++ b/manifests/j/JReleaser/jreleaser/1.18.0/JReleaser.jreleaser.installer.yaml
@@ -1,0 +1,28 @@
+# Generated with JReleaser 1.18.0-SNAPSHOT at 2025-04-30T09:23:20.225786408Z
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: JReleaser.jreleaser
+PackageVersion: 1.18.0
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+InstallModes:
+  - silent
+  - silentWithProgress
+UpgradeBehavior: install
+Commands:
+  - jreleaser
+ProductCode: '{2F3CE50F-22AB-3CDE-B7A0-0A0F597FBDC8}'
+ReleaseDate: 2025-04-30
+AppsAndFeaturesEntries:
+- ProductCode: '{2F3CE50F-22AB-3CDE-B7A0-0A0F597FBDC8}'
+  UpgradeCode: '{3D0E8D15-CF1B-32B1-B479-86F8424F8C46}'
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%/jreleaser'
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/jreleaser/jreleaser/releases/download/v1.18.0/jreleaser-installer-1.18.0-windows-x86_64.msi
+    InstallerSha256: eceb9de6992d6b90a246b42350423fc2ad48db003e058244e8e86831100463b9
+    ProductCode: 'jreleaser 1.18.0'
+ManifestType: installer
+ManifestVersion: 1.9.0

--- a/manifests/j/JReleaser/jreleaser/1.18.0/JReleaser.jreleaser.locale.en-US.yaml
+++ b/manifests/j/JReleaser/jreleaser/1.18.0/JReleaser.jreleaser.locale.en-US.yaml
@@ -1,0 +1,119 @@
+# Generated with JReleaser 1.18.0-SNAPSHOT at 2025-04-30T09:23:20.225786408Z
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: JReleaser.jreleaser
+PackageVersion: 1.18.0
+PackageLocale: en-US
+Publisher: JReleaser
+PublisherUrl: https://jreleaser.org
+PublisherSupportUrl: https://github.com/jreleaser/jreleaser/issues
+Author: Andres Almiray
+PackageName: jreleaser
+PackageUrl: https://jreleaser.org
+License: Apache-2.0
+LicenseUrl: https://spdx.org/licenses/Apache-2.0.html
+Copyright: 2020 Andres Almiray
+ShortDescription: Release projects quickly and easily with JReleaser
+Description: |
+  JReleaser is a release automation tool. Its goal is to simplify creating releases and
+  publishing artifacts to multiple package managers while providing customizable options.
+  
+  JReleaser takes inputs from popular builds tools (Ant, Maven, Gradle) such as JAR files,
+  binary distributions (.zip, .tar), JLink images, or any other file that you’d like to
+  publish as a Git release on popular Git services such as GitHub, GitLab, or Gitea.
+  Distribution files may additionally be published to be consumed by popular package managers
+  such as Homebrew, Chocolatey, Snapcraft, or get ready to be launched via JBang. Releases
+  may be announced in a variety of channels such as Twitter, Zulip, SDKMAN!, and more.
+Moniker: jreleaser
+Tags:
+  - git
+  - java
+  - jreleaser
+  - release
+  - tool
+ReleaseNotes: |-
+  Binaries
+  https://github.com/jreleaser/jreleaser/wiki/Release-v1.18.0
+  Changelog
+  🚀 Features
+  assemble
+  - e25f9e9 Native-image assembler can assemble FLAT_BINARY distributions, closes #1841
+  build
+  - 474ca0c Add git to docker images, closes #1838
+  deploy
+  - 9367144 Increase delay/retries in Maven deployers
+  - 148e2bd Support deploying to Nexus3, closes #1836
+  - ec179c4 Warn when deployment to Maven Central times out, closes #1826
+  packagers
+  - b2b8098 Support podman command for building container images, closes #1816
+  release
+  - 3e49e05 Let github.release.makeLatest be specified as input, closes #1847
+  - 9edf164 Support Forgejo as releaser, closes #1842 #1843
+  🐛 Fixes
+  announce
+  - 0c9352a Skip relocating jakarta.mail as it causes CNFE with smtp, closes #1860
+  assemble
+  - 25399ed Only resolve selected assemblers, closes #1828
+  core
+  - 7372af1 Normalize packager names for inclusion/exclusion, closes #1829
+  - da12d7c Validate default docker registry, closes #1827
+  deploy
+  - 5e1bd4e Fix calculation of deployable coordinates for snapshot artifacts, closes #1848
+  - 65a0891 Add missing Forgejo messages, closes #1842
+  packager
+  - a6d0463 Update AppImage templates, closes #1821
+  🛠  Build
+  - 6a2e5b4 Upgrade to Gradle 8.14
+  - 3dd7c08 Update release announcements
+  - afdace7 Update jreleaser-setup.sh
+  - 75b1b7d Fix readme
+  - 0438e51 Add link to Reproducible Central badge
+  - 46c7ae7 Skip milestone & release candidates for app-image metadata
+  📝 Documentation
+  - c28ddbf Add agreedSkiing as a contributor for code
+  - 7daa791 Add hboutemy as a contributor for infra
+  - 6a4f8f7 Add preimers as a contributor for code
+  - c6579db Add martinfrancois as a contributor for code
+  - df98677 Add anneloreegger as a contributor for code
+  ⚙️  Dependencies
+  - 7334cd8 Update asm to 9.8
+  - 1d74345 Update aws-java-sdk to 2.31.30
+  - 525f448 Update byte-buddy to 1.17.5
+  - 46ce530 Update checker-qual to 3.49.2
+  - eaabeae Update checkstyle to 10.23.0
+  - 731b8e3 Update classworlds to 2.9.0
+  - f663502 Update commons-io to 2.19.0
+  - 6271a8b Update commons-jexl to 3.5.0
+  - a6966bf Update commons-text to 1.13.1
+  - 8f1f9df Update cosign to 2.5.0
+  - baacbb3 Update errorprone to 2.38.0
+  - 698aa23 Update graalvm to 22.3.5
+  - 28393ba Update jackson to 2.19.0
+  - 801c2f1 Update jacoco to 0.8.13
+  - 11d1973 Update jib to 0.13.0
+  - db19f2b Update jsonschema to 4.38.0
+  - 20a1c6c Update json-smart to 2.5.2
+  - 3a45a10 Update jsoup to 1.19.1
+  - d7423be Update junit5 to 5.12.2
+  - 6bf76ea Update jakarta.mail-api to 2.1.3, closes #1861
+  - 53873e7 Update mockito to 5.17.0
+  - b0d83a9 Update mojoexecutor to 2.4.1
+  - d21217d Update picocli to 4.7.7
+  - bfa6f7b Update slf4j to 2.0.17
+  - e0463aa Update spotbugs to 4.9.3
+  - 7972c8f Update syft to 1.23.1
+  - a04d1b0 Update zstd-jni to 1.5.7-2
+  - 33048ea Releasing version 1.18.0
+  - 6b4504e Bump for next development cycle
+  Contributors
+  We'd like to thank the following people for their contributions:
+  - Andres Almiray (@aalmiray)
+  - Annelore Egger
+  - Erik Åsén (@agreedSkiing)
+  - Francois Martin (@martinfrancois)
+  - Hervé Boutemy
+  - Ixchel Ruiz
+  - Patrick Reimers
+ReleaseNotesUrl: https://github.com/jreleaser/jreleaser/releases/tag/v1.18.0
+ManifestType: defaultLocale
+ManifestVersion: 1.9.0

--- a/manifests/j/JReleaser/jreleaser/1.18.0/JReleaser.jreleaser.yaml
+++ b/manifests/j/JReleaser/jreleaser/1.18.0/JReleaser.jreleaser.yaml
@@ -1,0 +1,8 @@
+# Generated with JReleaser 1.18.0-SNAPSHOT at 2025-04-30T09:23:20.225786408Z
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: JReleaser.jreleaser
+PackageVersion: 1.18.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.9.0

